.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 212
+
+[lib/dependencies/AMDDefineDependency.js]
+max_line_length = 483
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
markdown uses `  ` (two spaces) at the end of a line  
to insert a line break in the middle of a paragraph.